eva-accession-clustering/src/test/java/uk/ac/ebi/eva/accession/clustering/configuration/batch/steps/ClusteringVariantStepConfigurationTest.java

```diff
@@ -246,7 +246,7 @@ private void assertGeneratedAccessions(String collectionName, String accessionFi
 
     private void assertSubmittedVariantsUpdated() {
         assertTrue(allSubmittedVariantsClustered());
-        List<Long> expectedAccessions = Arrays.asList(3000000000L, 3000000000L, 3000000001L, 3000000002L, 3000000003L);
+        List<Long> expectedAccessions = Arrays.asList(3000000000L, 3000000001L, 3000000002L, 3000000002L, 3000000003L);
```

Comment:
rsids assigned change slightly because of the change in the order that variants are read in